README.md

```diff
@@ -130,13 +131,6 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [terminal_table.cr](https://github.com/benoist/terminal_table.cr) - Simple ASCII table generator
  * [todo](https://github.com/Nephos/todo) - Todo list working in command line
 
-## Configuration
- * [cr-dotenv](https://github.com/gdotdesign/cr-dotenv) - Loads .env file
```

Comment:
`habitat` yes, I missed that. about `dotenv` I'm not sure... it's a data format and mostly used for configuration, but that's similar for e.g. INI or YAML.

Reply:
`dotenv` is mostly used for application configuring, honestly I'm not aware of any other use case.

Reply:
Dotenv is never really parsed into a datastructure, the results of parsing are always immediately sent to `ENV`. So it's much more of a configuration library than a dataformat, since the dataformat is always used for one thing.